### PR TITLE
Reverts undocumented change that makes projectiles much faster and go further

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

https://github.com/yogstation13/Yogstation/pull/19151

# Changelog

:cl:  
bugfix: Fixed projectiles being a lot faster and going a lot further than intended
/:cl:

### CODE REVIEW

**Review by ToasterBiome — APPROVED** [review]

**Comment by iamthesilly** [comment]
Oh god I never got rid of that change

I was dealing with some weird projectile behavior when making that PR like bullets passing through bodies so tried messing around with the projectile system to fix it, I did not mean to leave that change there

In conclusion: please merge this PR I did not mean to do this

**Comment by TheRyeGuyWhoWillNowDie** [comment]
> Oh god I never got rid of that change
> 
> I was dealing with some weird projectile behavior when making that PR like bullets passing through bodies so tried messing around with the projectile system to fix it, I did not mean to leave that change there
> 
> In conclusion: please merge this PR I did not mean to do this

local yogstation merges prs without actually reading them
tl;dr why am i not a maintainer yet
